plugins/ks.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ks
```

Comment:
The name is short, but fine I guess :)

Reply:
Short is good. Because we don't need to type too many words.